monitoring/uss_qualifier/scd/test_definitions/CHE/automated_test/automated_test_2.json

```diff
@@ -0,0 +1,116 @@
+{
+    "name": "Flight Authorisation validation test",
```

Comment:
If this is a draft of the the flight authorisation data format test, could we put it in `CHE/u-space/flight-authorisation-validation.json` to mirror [the documentation](https://github.com/interuss/dss/tree/master/interfaces/automated-testing/scd/design/u-space/flight-authorisation-validation)?